ocaml/xapi/xapi_host_helpers.ml

```diff
@@ -295,6 +295,9 @@ let startup_complete_m = Mutex.create ()
 let signal_startup_complete () =
   with_lock startup_complete_m (fun () -> startup_complete := true)
 
+let is_startup_complete () =
+  with_lock startup_complete_m (fun () -> !startup_complete)
```

Comment:
Why does this need a lock? It only reads a value

Reply:
Are operations on a boolean variable in OCaml always atomic?

Reply:
I see it is read with lock this way: https://github.com/xapi-project/xen-api/blob/master/ocaml/xapi/xapi_host_helpers.ml#L300